### DISCUSSION TITLE
Why doesn't Laravel's DynamoDb Cache Driver support tags?

### DISCUSSION THREAD

**Discussion by davefv** [laravel/framework#40537] [General]
I think it could be very useful to have tag support in the DynamoDB Cache Driver. For example to use it on Laravel Vapor without the need to pay for a NAT Gateway for the Elasticache instance.
I am thinking of implementing a driver with tag support, but I would like to know the reasons why this option is discarded in Laravel (complexity, performance or whatever...)

**Comment by bradley-r-martin** [comment]
@davefv Did you end up implementing a DynamoDb driver with support for tags ?

**Reply by davefv** [reply]
Unfortunately, I made a first attempt without success. My idea was to use the primary key and the sort key of the table to store both the cache and its relations with the tags, but I got lost in the complexity of the implementation :(
